res/controllers/lib/mididispatcher.mjs

```diff
@@ -0,0 +1,26 @@
+export class MidiDispatcher {
+    constructor(noteOff) {
+        this.noteOff = noteOff;
+        this.inputMap = new Map();
```

Comment:
It’s not clear whether direct access to inputMap is allowed or not. If not, how would unbinding callbacks be handled?